scripts/zones/Metalworks/npcs/Izabele.lua

```diff
@@ -8,7 +8,6 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    player:startEvent(745)
```

Comment:
It looks like Izabele needs a new entry in the DefaultActions?

Reply:
Don't think so! She only offers this quest, and then gains the new defaulAction upon this quest's completion